common/artifacts/999_wc_artifacts_other_consumables.txt

```diff
@@ -0,0 +1,29 @@
+# Food, potions, alcohol
```

Comment:
Would be nice if you see what you will get here.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103140103-0ab0e480-46fc-11eb-9900-086f9adb3b0d.png)

</details>

Reply:
you mean what modifier it gives?

Reply:
> you mean what modifier it gives?

Yes.